src/main/scala/org/apache/spark/sql/suites/CacheMissSuite.scala

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.suites
+
+import org.apache.spark.sql.{BenchmarkConfig, CacheMissConfigSet, OapTestSuite}
+
+object CacheMissSuite extends OapTestSuite with CacheMissConfigSet {
+
+  val table = "store_sales"
+
+  val attr = "ss_customer_sk"
+
+  private def databaseName = {
+    val conf = activeConf
+    conf.getBenchmarkConf(BenchmarkConfig.FILE_FORMAT) match {
+      case "parquet" => "parquet_tpcds_200"
```

Comment:
I updated some code, please use 
OapBenchmarkDataBuilder.getDatabase(activeConf.getBenchmarkConf(BenchmarkConfig.FILE_FORMAT))

Reply:
Ok,update it later